assets/src/components/dev-tools-toggle/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * WordPress dependencies
+ */
+import { useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { AMPSettingToggle } from '../amp-setting-toggle';
+import { User } from '../../components/user-context-provider';
```

Comment:
Import can be shortened.

```suggestion
import { User } from '../user-context-provider';
```